README.md

```diff
@@ -1,51 +1,127 @@
 # Google Fonts collection to WordPress Font Library collection
 
-
-This is a prototype repo to test some ideas for font preview generation for the new WordPress Font Library.
+Node scripts to generate the Google Fonts collection json and associated font previews for the WordPress Font Library.
```

Comment:
```suggestion
This repo contains Node scripts to generate the Google Fonts collection JSON and associated font previews for the WordPress Font Library.

Here are examples of the JSON files these scripts generate:

- https://s.w.org/images/fonts/17.7/collections/google-fonts.json
- https://s.w.org/images/fonts/17.7/collections/google-fonts-with-preview.json
```

Reply:
Thanks! Added in https://github.com/WordPress/google-fonts-to-wordpress-collection/pull/32/commits/1ccf7171d03f3c200a4fb6fbafb8831f8a7333ee